presentation/common/src/presentation-common/content/Descriptor.ts

```diff
@@ -356,8 +395,12 @@ export class Descriptor implements DescriptorSource {
       overrides.displayType = this.displayType;
     if (this.contentFlags !== 0)
       overrides.contentFlags = this.contentFlags;
-    if (this.filterExpression)
-      overrides.filterExpression = this.filterExpression;
+    if (this.filterExpression || this.fieldsFilterExpression) { // eslint-disable-line deprecation/deprecation
+      overrides.filterExpression = this.filterExpression ?? this.fieldsFilterExpression; // eslint-disable-line deprecation/deprecation
+      overrides.fieldsFilterExpression = this.fieldsFilterExpression ?? this.filterExpression; // eslint-disable-line deprecation/deprecation
+    }
```

Comment:
I don't think I understand what this is doing. Why not just assign descriptor's `filterExpression` and `fieldsFilterExpression` to same attributes of `overrides`?